lib/kubeclient.rb

```diff
@@ -287,6 +287,7 @@ def define_entity_methods
           # This method used to take resource_version as a param, so
           # this conversion is to keep backwards compatibility
           options = { resource_version: options } unless options.is_a?(Hash)
+          options[timeoutSeconds] = 380 # 6 mins 20 seconds
```

Comment:
Did you mean `:timeoutSeconds` ?
This will override any passed-in timeout.  I'm assuming you're just experimenting for now, but this can't be merged.